src/System.Device.Gpio/System/Device/Pwm/Drivers/Windows10PwmDriverChannel.Windows.cs

```diff
@@ -26,8 +26,8 @@ public void ChangeDutyCycle(double dutyCyclePercentage)
 
         public void Start(double dutyCyclePercentage)
         {
-            this.ChangeDutyCycle(dutyCyclePercentage);
             _winPin?.Start();
```

Comment:
The behavior I'm seeing is that there is no output at all until both Start() and then SetActiveDutyCyclePercentage() are called. SetActiveDutyCyclePercentage() could be called before Start() as well to ensure the initial dutycycle is correct.

Reply:
So...why are we making this change?  Is it because it doesn't work on Hummingboard without it?


---
In reply to: [241897230](https://github.com/dotnet/iot/pull/97#discussion_r241897230) [](ancestors = 241897230)